plugins/coverbrowser.koplugin/main.lua

```diff
@@ -294,6 +298,74 @@ function CoverBrowser:addToMainMenu(menu_items)
                         end,
                     },
+                    {
+                        text = _("Series "),
+                        sub_item_table = {
+                            {
+                                text = _("Append series metadata to authors"),
+                                checked_func = function() return series_mode == "append_series_to_authors" end,
+                                callback = function()
+                                    if series_mode == "append_series_to_authors" then
+                                        series_mode = nil
+                                    else
+                                        series_mode = "append_series_to_authors"
+                                    end
+                                    BookInfoManager:saveSetting("series_mode", series_mode)
+                                    self:refreshFileManagerInstance()
+                                end,
+                            },
+                            {
+                                text = _("Append series metadata to title"),
+                                checked_func = function() return series_mode == "append_series_to_title" end,
+                                callback = function()
+                                    if series_mode == "append_series_to_title" then
+                                        series_mode = nil
+                                    else
+                                        series_mode = "append_series_to_title"
+                                    end
+                                    BookInfoManager:saveSetting("series_mode", series_mode)
+                                    self:refreshFileManagerInstance()
+                                end,
+                            },
+                            {
+                                text = _("Show series metadata in separate line"),
+                                checked_func = function() return series_mode == "series_in_separate_line" end,
+                                callback = function()
+                                    if series_mode == "series_in_separate_line" then
+                                        series_mode = nil
+                                    else
+                                        series_mode = "series_in_separate_line"
+                                    end
+                                    BookInfoManager:saveSetting("series_mode", series_mode)
+                                    self:refreshFileManagerInstance()
+                                end,
+                            },
+                        },
+                        separator = true
+                    },
+                    {
+                        text = _("(Detailed list) Files per page"),
+                        help_text = _([[This sets the number of files and directories
+                        per page in non-'classic' display modes.]]),
```

Comment:
This should be on a single line - possibly wrapped the following way if you feel it's needed:
https://github.com/koreader/koreader/blob/65fcd385c01f6eb6a1bab7d3404899938c04eaa1/frontend/apps/reader/modules/readerlink.lua#L134-L136
(The leading space is removed, but that's the only stuff removed - your big amount of spaces in the middle would stay there, and show up to translators.)

Reply:
So if I were to go with max 100 characters per line, I would use string concatenation?

Reply:
No, never. The line should be as long as it needs to be. If it feels too long, rewrite it to be shorter.

(But this line is only 86 characters long…?)

Reply:
Minor correction: xgettext does happen to support the practice, so since #5237/#5242 it's technically possible.

However, the fact remains however that I'm opposed to abiding by line length limits like some kind of machine, especially if it results in ugly string concatenation.

For code it's a fairly strong indicator that you might want to consider refactoring, but text isn't code. At the same time, for text it can also an indicator that you might want to consider rewriting for clarity and brevity depending on context. But of course only if you're talking about line length within the string, not indentation plus string length.

As @poire-z said, for longer strings it works better to do:
```lua
                                some_string = [[
This is the string.]]
```